src/OrchardCore/OrchardCore.Abstractions/Json/Nodes/JArray.cs

```diff
@@ -118,7 +118,7 @@ public static bool ContainsValue(this JsonArray? jsonArray, JsonValue? value)
 
         settings ??= new JsonMergeSettings();
 
-        switch (settings?.MergeArrayHandling ?? MergeArrayHandling.Concat)
+        switch (settings.MergeArrayHandling)
```

Comment:
> The only thing that can be safely removed is the ? after settings, since ln 119 takes care of settings being null.

But the `MergeArrayHandling.Concat` will no longer been executed

Reply:
FYI `MergeArrayHandling.Contact` is default value for the enum, so I don't think there's an issue here

Reply:
Because you should test every change, even if it seems obvious to you, since it might break things in non-obvious ways.

Reply:
The unit tests should confirm that change is using the concatination by default

Reply:
Great, thank you!